Corgibytes.Freshli.Cli.Test/Corgibytes.Freshli.Cli.Test.csproj

```diff
@@ -17,6 +17,10 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
     <PackageReference Include="Moq" Version="4.18.1" />
     <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
+    <PackageReference Include="SonarAnalyzer.CSharp" Version="8.39.0.47922">
```

Comment:
I chose SonarLint because it came highly recommended, and the existing linters were not catching many of the things it did (to wit, most of the issues addressed in this commit).